frontend/src/scenes/insights/ActionFilter/renameModalLogic.test.ts

```diff
@@ -0,0 +1,88 @@
+import { BuiltLogic } from 'kea'
+import {
+    BareEntity,
+    entityFilterLogic,
+    EntityFilterProps,
+    LocalFilter,
+} from 'scenes/insights/ActionFilter/entityFilterLogic'
+import { expectLogic, initKeaTestLogic } from '~/test/kea-test-utils'
+import filtersJson from './__mocks__/filters.json'
+import { EntityFilter } from '~/types'
+import { renameModalLogicType } from 'scenes/insights/ActionFilter/renameModalLogicType'
+import { renameModalLogic, RenameModalProps } from 'scenes/insights/ActionFilter/renameModalLogic'
+import { mockAPI } from 'lib/api.mock'
+import { entityFilterLogicType } from 'scenes/insights/ActionFilter/entityFilterLogicType'
+import { getDisplayNameFromEntityFilter } from 'scenes/insights/utils'
+
+jest.mock('lib/api')
+
+describe('entityFilterLogic', () => {
+    let logic: BuiltLogic<renameModalLogicType<RenameModalProps>>
+    let entityLogic: BuiltLogic<entityFilterLogicType<BareEntity, EntityFilterProps, LocalFilter>>
+
+    mockAPI(async ({ pathname, searchParams }) => {
+        if (
+            ['api/action/', 'api/projects/@current/event_definitions/', 'api/users/@me/', 'api/dashboard'].includes(
+                pathname
+            )
+        ) {
+            return { results: [] }
+        } else {
+            throw new Error(`Unmocked fetch to: ${pathname} with params: ${JSON.stringify(searchParams)}`)
+        }
+    })
+
+    initKeaTestLogic({
```

Comment:
are you thinking so that you can mock a dependent logic?

would mocking the module in the test file do it?

Reply:
What exactly do you want to mock in/out of the other logic? Nothing exists for this now, though we can always add things. For example, it could be possible to add code that would prevent any listeners running on an adjacent logic, or to make actions do nothing, or ... 

However this is fine as well I think.

Reply:
Sorry I should've clarified more-- I was wondering if there's a way to mock the actions in `entityFilterLogic` that a reducer in `renameModalLogic` depends on. I couldn't get around this without initializing the entire `entityFilterLogic` with props because it was throwing some "accessing {property} of undefined errors". It just seems a little awkward to setup and run logic B code in a logic A test—having a util function that noops on all actions of a mock could be a nice addition to the kea test suite that I can pickup if I have some time. 

For now I'll keep this as it is